src/main/java/io/github/liias/monkey/project/sdk/SdkHelper.java

```diff
@@ -0,0 +1,32 @@
+package io.github.liias.monkey.project.sdk;
+
+import com.intellij.execution.configurations.RuntimeConfigurationException;
+import com.intellij.openapi.projectRoots.Sdk;
+import com.intellij.openapi.util.SystemInfoRt;
+import org.jetbrains.annotations.NotNull;
+
+public abstract class SdkHelper {
+	protected final Sdk sdk;
+
+	SdkHelper(Sdk sdk) {
+		this.sdk = sdk;
```

Comment:
As sdk is unused here I think this helper class could be simpler if having only static methods and no subclasses.